pkg/compose/compose.go

```diff
@@ -211,8 +212,11 @@ func (s *composeService) projectFromName(containers Containers, projectName stri
 					if len(dcArr) > 2 {
 						restart, _ = strconv.ParseBool(dcArr[2])
 					}
+					if len(dcArr) > 3 {
```

Comment:
this block is for backward compatibility only, I don't think we should extend syntax to support a 3rd argument

Reply:
I'm definitely not a big fan of stuffing comma&colon-delimited data into the label, but without it here, what does that mean? When in project mode, we'd consider all dependencies required? 🤔